webpack/assets/javascripts/react_app/components/Layout/Layout.fixtures.js

```diff
@@ -190,15 +191,6 @@ const organizations = {
     { id: 1, title: 'org1', href: '/organizations/1-org1/select' },
     { id: 2, title: 'org2', href: '/organizations/2-org2/select' },
   ],
-  many_organizations: [
-    { id: 1, title: 'org1', href: '/organizations/1-org1/select' },
```

Comment:
just making sure, it wasn't used?

Reply:
This was used to test the `TaxonomyDropdown` search. Now that the search is in pf I removed the test